Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -11,9 +11,17 @@ param(
     })] 
     [string] $installConfigPath = "$PSScriptRoot\install.config", 
     [switch] $noDiscoveryService, 
-    [switch] $quiet
+    [switch] $quiet,
+    [ValidateScript({
+        if (!(Test-Path $_)) {
+            throw "Path $_ does not exist. Please enter valid path to the IdentitySearchProvider web.config."
+        }
+        return $true
+    })]
+    [string] $idpssDirectoryPath
```

Comment:
I think we should either get this path by interogating IIS (we do this in the register.ps1 script), or have it as an install.config variable. We want to stick to the standard paramters on the script.